tomviz/modules/ModuleSlice.cxx

```diff
@@ -97,6 +97,9 @@ bool ModuleSlice::initialize(DataSource* data, vtkSMViewProxy* vtkView)
   controller->PostInitializeProxy(m_passThrough);
   controller->RegisterPipelineProxy(m_passThrough);
 
+  // We are calling this so that the "Reset Center" button will be enabled
+  controller->Show(m_passThrough, 0, vtkView);
```

Comment:
This creates an outline representation by default.  So if you turn off the visibility of the outline module (or delete it), you still have the outline from this.

Reply:
Hmm... yeah I see that. I will look into that.